Microsoft.Xrm.Data.PowerShell/Microsoft.Xrm.Data.PowerShell.psm1

```diff
@@ -3943,6 +3943,74 @@ function Invoke-CrmWhoAmI{
     return $result
 }
 
+function Invoke-CrmAction {
+# .ExternalHelp Microsoft.Xrm.Data.PowerShell.Help.xml
+    [OutputType([hashtable])]
+    [OutputType([Microsoft.Xrm.Sdk.OrganizationResponse], ParameterSetName="Raw")]
+    param (
+        [Microsoft.Xrm.Tooling.Connector.CrmServiceClient]
+        $conn,
+
+        [Parameter(
+            Position=1,
+            Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $Name,
+
+        [Parameter(ValueFromPipeline)]
+        [ValidateNotNullOrEmpty()]
+        [Microsoft.Xrm.Sdk.EntityReference]
+        $Target,
+
+        [Parameter(Position=2)]
+        [ValidateNotNullOrEmpty()]
+        [hashtable]
+        $Parameters,
+
+        [Parameter(ParameterSetName="Raw")]
+        [switch]
+        $Raw
```

Comment:
I don't think -Verbose could fully display all the details of each type of returned value. I just found it useful to have the alternate option of getting the raw response rather than accessing through the hashtable. It's something I think would be useful in these types of functions that are wrapping another API, and think could be done in other areas too. If you really don't want it I think I'd just rather remove it.